### DISCUSSION TITLE
Remove unused images

### DISCUSSION THREAD

**Discussion by mikednz** [timber/timber#2864] [Q&A]
Hi everyone. I have a lot of images being uploaded and using Timber to create some crops on the fly that I can see are saved in my uploads folder. I want to remove unused images as I have been running the site for over 10 years.

I can see when using the |resize(400,400) twig helper the image ending file-400x400-c-default.jpg for example are added to my uploads folder. 

I wanted to safely use a plugin like https://wordpress.org/plugins/media-cleaner/ for example but I'm not sure if the file-400x400-c-default.jpg is the one that is added to the database and will not be deleted?

Had anyone had any experience using a plugin to remove unused images with Timber?

**Comment by cbirdsong** [comment]
I know this is more than a year old, but if you're still having this issue I would try using Media Cleaner on a staging environment, but running a crawl of the site using a tool like [Screaming Frog](https://www.screamingfrog.co.uk) before and after you do it. It might require some special configuration to make sure it checks every size of every image that it finds, but that would tell you if any broken images were created, and you could probably also compare the list of images it discovers against a list created by crawling through the file system directly.
